lib/airbrussh/formatter.rb

```diff
@@ -109,14 +109,20 @@ def write_command(command)
       ctx = context_for_command(command)
       number = '%02d' % ctx.number
 
-      if ctx.first_execution?
-        description = yellow(ctx.shell_string)
-        print_line "      #{number} #{description}"
-      end
-
-      if command.finished?
-        status = format_command_completion_status(command, number)
-        print_line "    #{status}"
+      if ctx.first_execution? || command.finished?
+        if ctx.first_execution?
+          description = yellow(ctx.shell_string)
+          print_line "      #{number} #{description}"
+        end
+
+        if command.finished?
+          status = format_command_completion_status(command, number)
+          print_line "    #{status}"
+        end
+      elsif Airbrussh.configuration.log_level == :debug
+        command.full_stdout.split("\n").each do |line|
+          print_line "      #{number} #{line}"
+        end
```

Comment:
@carlesso I believe that after this, you need to manually clear `command.full_stdout`. The command gets logged every time SSHKit receives a new chunk of data, and each time it is appending that data to `full_stdout`.

You can see that SSHKit is doing that here: https://github.com/capistrano/sshkit/blob/v1.7.1/lib/sshkit/formatters/pretty.rb#L37